scripts/deploySidechain.ts

```diff
@@ -300,88 +355,52 @@ export async function deploySidechainSystem(
     tx = await booster.setFees(550, 1100, 50, 0);
     await waitForTx(tx, debug, waitForBlocks);
 
-    tx = await booster.setFeeManager(addresses.daoMultisig);
+    tx = await booster.setFeeManager(multisigs.daoMultisig);
     await waitForTx(tx, debug, waitForBlocks);
 
     tx = await booster.setOwner(boosterOwner.address);
     await waitForTx(tx, debug, waitForBlocks);
 
-    return {
-        voterProxy,
-        booster,
-        boosterOwner,
-        factories: {
-            rewardFactory,
-            stashFactory,
-            tokenFactory,
-            proxyFactory,
-        },
-        poolManager,
-        auraOFT,
-        l2Coordinator,
-    };
-}
-
-export interface SidechainMocksDeployed {
-    token: MockERC20;
-    bpt: MockBalancerPoolToken;
-    minter: MockCurveMinter;
-    gauge: MockCurveGauge;
+    return contracts;
 }
 
-export async function deploySidechainMocks(
-    hre: HardhatRuntimeEnvironment,
+export async function deploySidechainPhase2(
```

Comment:
There is only 1 phase currently. No need to add this as there is no gauge controller etc on the L2s.